agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -210,6 +210,7 @@ class backhaul_manager : public btl::transport_socket_thread {
     const int DEAUTH_REASON_PASSPHRASE_MISMACH        = 2;
     const int AUTOCONFIG_DISCOVERY_TIMEOUT_SECONDS    = 1;
     const int MAX_FAILED_AUTOCONFIG_SEARCH_ATTEMPTS   = 20;
+    const int DISCOVERY_NOTIFICATION_TIMEOUT_SECONDS  = 63; // 60 sec (standard) + 3 sec grace
```

Comment:
(nitpick) I'd call it something like `DISCOVERY_REMOVAL_TIMEOUT_SECONDS` because it's more about when you consider the node as down than about sending the notification.

Reply:
I don't know about it...  Notification timeout seems right to me.

Reply:
And then use DISCOVERY_NOTIFICATION_TIMEOUT_SEC instead of the hardcoded 60 in the backhaul manager to send out the discovery.

Reply:
ok